deps/GraphBLAS/Config/GraphBLAS.h.in

```diff
@@ -5568,11 +5568,11 @@ GrB_Info GxB_MatrixTupleIter_new
     GrB_Matrix A                // matrix being iterated
 ) ;
 
-// Iterate over a specific column
-GrB_Info GxB_MatrixTupleIter_iterate_column
+//iterate over specific index
+GrB_Info GxB_MatrixTupleIter_iterate
 (
-    GxB_MatrixTupleIter *iter,       // iterator to use
-    GrB_Index colIdx        // column index to iterate over
+    GxB_MatrixTupleIter *iter,
```

Comment:
GxB_MatrixTupleIter *iter, // iterator to use

Reply:
done